testbed/pyproject.toml

```diff
@@ -1,8 +1,20 @@
-# This project was generated using template: https://github.com/beeware/briefcase-template and branch: v0.3.12
+[project]
+name = "testbed"
+version = "0.0.1"
+dependencies = ["../core"]
+
+[project.optional-dependencies]
```

Comment:
Nice hack - it's obvious now I think about it, but it hadn't occurred to me to do this.